xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/main/java/org/xwiki/livedata/LiveDataException.java

```diff
@@ -45,6 +45,19 @@ public LiveDataException(Throwable source)
         super(source);
     }
 
+    /**
+     * Constructs a new live data exception with the specified detail message.
+     *
+     * @param message the detail message
+     * @since 13.2RC1
```

Comment:
Of course the versions are outdated but I'll update them as soon as we are ok for the rest.